# Project suggestions


* RNA-seq analysis with cell type deconvolution: Follow the approach [here](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008120) and analyze an RNA-seq dataset generated at the FGCZ, more infos from Hubert,  https://sta426hs2020.slack.com/archives/D01AE6J7EKD
* Collaborative project (with Mark and a group from USZ): Analysis of single cell RNA-seq data: 1) to determine a suitable list of genes for in situ experiments; 2) integrating the local data with public datasets to find signatures of drug resistance (could actually be two projects)
* Methods project: understanding whether gene selection has an impact on performance for [SingleR](https://bioconductor.org/packages/release/bioc/html/SingleR.html) cell type prediction
* Collaborative project (with Mark and a group from USZ): Analysis of single cell CITE-seq data: goals to be determined (roughly, do QC of the data, determine cell types, compare cell type compositions between normal, precursor, cancer cells)
